app/services/course/assessment/reminder_service.rb

```diff
@@ -32,9 +33,10 @@ def closing_reminder(assessment, token)
 
   def uncompleted_students(assessment)
```

Comment:
Could you add the params that `uncompleted_students` return? Would be easier to modify in the future.